add comprehensive colorscheme screenshots

favorable-mutation opened the pull request:
Currently, there are some screenshots of available colorschemes at the beginning of the README.

These changes add a new `.md` file linked to from the README with a comprehensive set of screenshots for all available colorschemes, with the image files linked from the project Wiki using relative paths.

I have all of the screenshots in the Wiki of my fork, so feel free to pull that down with `git clone https://github.com/favorable-mutation/lightline.vim.wiki.git` in order to move them over to the actual project's Wiki.

itchyny commented:
Okay, I think it's useful for users when they choose the colorscheme. One thing to point out, I prefer `colorscheme.md` rather than `Colors.md` for filename. Also I'm thinking of generating the screenshot automatically by script but that's my own homework :)

favorable-mutation commented:
Sounds good to me! Rebased off of `master`, renamed the file, and updated to include `simpleblack`. Lmk if there's anything else you'd like to see changes to, and thanks for reviewing!
